rsql-common/src/main/java/io/github/perplexhub/rsql/RSQLCustomPredicateInput.java

```diff
@@ -15,16 +16,18 @@ public class RSQLCustomPredicateInput {
     CriteriaBuilder criteriaBuilder;
     Path<?> path;
     Attribute<?, ?> attribute;
+    String attributeName;
     List<Object> arguments;
     From<?, ?> root;
 
     public static RSQLCustomPredicateInput of(
             CriteriaBuilder criteriaBuilder,
             Path<?> path,
+            String attributeName,
```

Comment:
It's breaking change. Could we do this in backward compatible manner?

Reply:
For sure!